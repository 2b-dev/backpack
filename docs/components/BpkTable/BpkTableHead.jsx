import React from 'react'
import CssModules from 'react-css-modules'

import styles from './bpk-table.scss'

const BpkTableHead = (props) => <thead {...props} />

export default CssModules(BpkTableHead, styles)
