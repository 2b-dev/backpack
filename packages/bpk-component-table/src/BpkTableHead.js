import React, { PropTypes } from 'react';

const BpkTableHead = props => <thead>{props.children}</thead>;

BpkTableHead.propTypes = {
  children: PropTypes.node.isRequired,
};

export default BpkTableHead;
