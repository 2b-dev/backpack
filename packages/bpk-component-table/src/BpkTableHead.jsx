import React, { PropTypes } from 'react'

const BpkTableHead = ({ children }) => <thead children={children} />

BpkTableHead.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired
}

export default BpkTableHead
