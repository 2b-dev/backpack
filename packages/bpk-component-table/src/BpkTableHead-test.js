import React from 'react'
import renderer from 'react/lib/ReactTestRenderer'
import BpkTableHead from './BpkTableHead'

describe('BpkTableHead', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <BpkTableHead>
        <th />
      </BpkTableHead>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
