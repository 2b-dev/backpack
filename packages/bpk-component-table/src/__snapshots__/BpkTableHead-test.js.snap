exports[`BpkTableHead should render correctly 1`] = `
<thead>
  <th />
</thead>
`;
